### DISCUSSION TITLE
Returning a video and playing it

### DISCUSSION THREAD

**Discussion by peterjbardenhagen** [microsoft/teams-sdk#2247] [Q&A]
I want to create a Teams chatbot, ideally in C# but open to Python or JS that can return a MP4 video.
I'm using Azure AI Search hybrid with an index of documents that can include MP4s.
If there is a match I want it to not only return the gen-AI result but a MP4 video as well and bonus points if it can skip through the video to the relevant part.
Is this possible, and if so any hints on how to make this work?

**Comment by heyitsaamir** [comment]
We're clearing discussions and starting fresh. This discussion has not been acted upon. Please reopen the discussion, or start a new one if this question is still important. Thank you for your understanding.
